pkg/controller/controller.go

```diff
@@ -302,15 +318,24 @@ func (s *Service) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest
 				if apiError, ok := err.(gopowerstore.APIError); ok && apiError.NotFound() {
 					log.Infof("Volume group with name %s not found, creating it", vgName)
 
+					// Attribute that indicates whether snapshot sets of the volumegroup will be write-order consistent.
+					isWriteOrderConsistent := false
```

Comment:
This will make write-order-consistent to be false for ASYNC mode while the API/UI's enables it by default for volume group creation.
CC @alankar-verma

Reply:
How about enabling it (sending true) irrespective of the mode?

Reply:
Hi @santhoshatdell Currently for async we are not passing is-write-consistent ... so by default the value is going as false due to boolean default value (though from API perspective by default it is true ). For Sync, it should be true always so used this variable to pass  as true.. Can we  discuss on this and handle as part of upcoming PR s ?